application/config/module.config.php

```diff
@@ -529,6 +529,7 @@
             'url' => Site\Navigation\Link\Url::class,
             'browse' => Site\Navigation\Link\Browse::class,
             'browseItemSets' => Site\Navigation\Link\BrowseItemSets::class,
+            'siteSwitcher' => Site\Navigation\Link\SiteSwitcher::class,
```

Comment:
Could you separate out the siteSwitcher stuff into a separate branch/PR? I can see how they kind of work together but for review purposes, I think we'd like it separate from the values feature.

Reply:
OK, SiteSwitcher removed in f72b394
I'll submit a separate PR.